### PR TITLE
Update ah2020 meeting page

### PR DESCRIPTION
**Pull request by AlasdairGray**
Adding links to slides and details of the eDGAR/PhenPath poster

### CODE REVIEW

File: _meetings/2020-06_ELIXIRAllHands2020.md [hunk 1]
```diff
@@ -17,10 +17,11 @@ attendees:
 
 During the ELIXIR All Hands Meeting 2020, the Interoperability Platform session will be dedicated to matters relating to the Bioschemas Strategic Implementation Study. Details of this session are in the [agenda document]({{agenda-doc}}).
 
-We have had two abstracts accepted for presentation at the ELIXIR All Hands Meeting 2020. The full abstracts are included below.
+We have had two abstracts and one poster accepted for presentation at the ELIXIR All Hands Meeting 2020. The full abstracts are included below.
 
 - [Exploiting Bioschemas Markup to Support ELIXIR Communities](#exploiting-bioschemas-markup-to-support-elixir-communities) accepted into the short talks session taking place on Wednesday 10 June at 10:00 (CEST)
 - [Discovering Biodiversity Resources on the Web](#discovering-biodiversity-resources-on-the-web) accepted into the [Biodiversity mini-symposium](https://docs.google.com/document/d/11NDVkT5baCmFfA0bUdIxCauiKhfQofQPiXkJsnVAbP0/edit?usp=sharing) taking place on Monday 8 June at 12:00 (CEST)
+- [eDGAR and PhenPath resources join the Bioschemas Community](#edgar-and-phenpath-resources-join-the-bioschemas-community)
```
Comment (ljgarcia) [inline _meetings/2020-06_ELIXIRAllHands2020.md:24]: This link is broken

File: _meetings/2020-06_ELIXIRAllHands2020.md [hunk 2]
```diff
@@ -44,3 +45,19 @@ Bioschemas is an approach to make life sciences resources more discoverable by e
 The Bioschemas community now has proposals for 17 types that extend the Schema.org vocabulary to enable it to be used by the life sciences community. These include types such as Taxon, BioSample, and Phenotype, that are directly relevant to the Biodiversity community. There are deployments of these at the National Museum of Natural History of Paris and the Botanical Collections at the Meise Botanic Garden, Belgium.
 
 To enable ELIXIR communities to exploit Bioschemas markup to populate community search portals, we have developed the B-MUSE directed crawler. B-MUSE will extract Bioschemas markup from a given set of pages or identifier pattern and make the structured content available for search. This approach has been demonstrated using the ELIXIR TeSS training portal, and is in active development in other ELIXIR communities, viz Rare Diseases, Plants, and Intrinsically Disordered Proteins.
+
+## eDGAR and PhenPath resources join the Bioschemas Community ([poster](https://f1000research.com/posters/9-569))
```
Comment (ljgarcia) [inline _meetings/2020-06_ELIXIRAllHands2020.md:49]: As the heading will be used an an internal link, maybe better to move the poster link to the corresponding text

File: _meetings/2020-06_ELIXIRAllHands2020.md (hunk 1 above)
Comment (AlasdairGray) [reply _meetings/2020-06_ELIXIRAllHands2020.md:24]: Fixed. I added the poster link in later and that broke the link

File: _meetings/2020-06_ELIXIRAllHands2020.md (hunk 2 above)
Comment (AlasdairGray) [reply _meetings/2020-06_ELIXIRAllHands2020.md:49]: Fixed the link with the poster in the title

**Pull request merged**